pkg/curatedpackages/packageinstaller_test.go

```diff
@@ -93,3 +93,40 @@ func TestPackageInstallerFailWhenPackageFails(t *testing.T) {
 
 	tt.command.InstallCuratedPackages(tt.ctx)
 }
+
+func TestPackageInstallerDisabled(t *testing.T) {
+	tt := newPackageInstallerTest(t)
+	tt.spec.Cluster.Spec.Packages = &anywherev1.PackageConfiguration{
+		Disable: true,
+	}
+
+	tt.packageClient.EXPECT().CreatePackages(tt.ctx, tt.packagePath, tt.kubeConfigPath).Return(nil)
+
+	tt.command.InstallCuratedPackages(tt.ctx)
+}
+
+func TestIsPackageControllerDisabled(t *testing.T) {
+	tt := newPackageInstallerTest(t)
+
+	if curatedpackages.IsPackageControllerDisabled(nil) == true {
```

Comment:
It is unnecessary to do equality check on booleans.

Reply:
Anyway, fixed